.swiftlint.yml

```diff
@@ -45,7 +45,6 @@ opt_in_rules:
   - override_in_extension
   - quick_discouraged_call
   - quick_discouraged_focused_test
-  - unused_import
```

Comment:
Sneak fix for linting warning due to misconfiguration of this rule.
This shouldn't be listed here.